src/dataimport/const.ts

```diff
@@ -95,6 +95,16 @@ export type TPDIOrderParams = {
   productKernel?: ResamplingKernel;
 };
 
+export type TPDIOrderCompatibleCollection = {
+  id: string;
```

Comment:
AIRBUS seems to also have `additionalData` field. I don't think it's very useful (like most of the fields), but it feels strange omitting a single field.

Reply:
I decided to remove everything except id and name as it is not being used